wordpress-mobile/WordPress-iOS
# Jetpack Overlay - Add dismiss button

Fixes #19177

This PR adds an "x" dismiss button to the top-trailing corner of the Jetpack overlay.
It also contains some layout and fonts adjustments, as well as an updated logo animation.

dark mode | light mode
-- | --
<img height = 390 src=https://user-images.githubusercontent.com/34376330/184376676-4dc0cf00-7dcc-416e-9d45-a0ee2a77b6fa.png> | <img height = 390 src=https://user-images.githubusercontent.com/34376330/184376708-bbe0b16c-32a0-40c9-bc30-988b97126c4b.png>

**Known issues: the landscape layout will be addressed in a separate PR**

To test:
- Checkout this branch and enable the `jetpackBottomSheet` feature flag
- Build/run
- Go to My Site/Home
- Tap the "Jetpack powered" badge
- Make sure the overlay appears, and contains a dismiss button on the top-trailing corner
- Tap that button, and make sure the overlay dismisses itself

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
